website/docs/reference/runtimes/python/index.mdx

```diff
@@ -36,4 +37,6 @@ def example_python(g: Graph):
 
 def say_hello(x: any):
     return f"Hello {x["name"]}"
-```
\ No newline at end of file
+```
+
+<!-- TODO: after python runtime rework lands -->
```

Comment:
Hey, so I haven't touched the frontned for the `PythonRuntime` on #687 . I think I'll manage to make it a non-breaking change. I have some breaking changes in mind but not in that PR.